AngularJS/angularjs-quiz.md

```diff
@@ -1,147 +1,153 @@
 ## AngularJs LinkedIn Assessment Questions Answers (Left)
 
-Q1. What is the purpose of the ViewChild decorator in this component class?
+#### Q1. What is the purpose of the ViewChild decorator in this component class?
 
+```
```

Comment:
if approach very strictly, for this code we have to use markdown with 

![image](https://user-images.githubusercontent.com/13170022/102257924-a4fb7600-3f0d-11eb-9e5b-0f5c218ce704.png)

to highlight code according to selected languages

![image](https://user-images.githubusercontent.com/13170022/102258452-3cf95f80-3f0e-11eb-8e7b-6ff7bb079d8e.png)

So does you Python script can identify what kind of languages  inside?